src/css/components/card.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../../node_modules/uswds/src/stylesheets/core/defaults.scss';
+@import '../override_vars.scss';
+@import '../core.scss';
+
+.card {
+  background-color: $color-white;
+  border: 1px solid $color-textblack;
+  border-radius: $rounded;
+  margin-bottom: $grid-2;
+  padding: 0;
+}
+
+.card-home-contents-title {
```

Comment:
tbh, i don't care. I can change them to be less specific, but i don't even know if these styles are in use at all. this is one of those things where I build something to try and show how it could work (as a prototype, essentially) and I don't think anything made it into production.

So sure, I'll change it.